interfaces/builtin/time_control.go

```diff
@@ -68,6 +68,13 @@ dbus (receive)
     member=PropertiesChanged
     peer=(label=unconfined),
 
+# Required in bionic (Ubuntu 18.04)
+dbus (send)
+    bus=system
+    path=/org/freedesktop/timedate1
+    interface=org.freedesktop.DBus.Properties
+    member=GetAll,
```

Comment:
This is meant to already be allowed by this rule:
```
# Read all properties from timedate1
dbus (send)
    bus=system
    path=/org/freedesktop/timedate1
    interface=org.freedesktop.DBus.Properties
    member=Get{,All}
    peer=(label=unconfined),
```
You dropped the peer label in this PR, which on its own is insufficient to address the underlying issue (eg, Get() would still fail). I suspect that this is a result of the DBus change where the service is DBus activated and its service file needs to be adjusted to have AssumedAppArmorLabel=unconfined, then you could drop this change.

Reply:
Nice catch - indeed - I filed https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1749000